public/app/plugins/panel/candlestick/CandlestickPanel.tsx

```diff
@@ -227,6 +217,20 @@ export const CandlestickPanel: React.FC<CandlestickPanelProps> = ({
     );
   }
 
+  if (shouldRenderPrice) {
```

Comment:
we can technically do this test by doing `!isEmptyObject(renderers[0].fieldMap)` instead of returning `shouldRenderPrice` since they follow same logic. but probably better to be explicit...but we can also just add a comment :)

Reply:
Do you want me to add a comment to it?

Reply:
no, let's just leave as-is.